index.js

```diff
@@ -40,17 +43,6 @@ module.exports = {
         "ignore": ["after-comment"]
       }
     ],
-    "scss/at-extend-no-missing-placeholder": true,
```

Comment:
All these rules stayed in the config, I've just moved them below the @stylistic rules